apps/etterlatte-saksbehandling-ui/client/src/components/manuelbehandling/ManuellBehandling.tsx

```diff
@@ -157,7 +162,30 @@ export default function ManuellBehandling() {
           ))}
         </Select>
 
-        <Checkbox {...register('overstyrBeregning')}>Skal bruke manuell beregning</Checkbox>
+        <Box paddingBlock="4 4">
+          <Checkbox
+            {...register('overstyrBeregning')}
+            onChange={(e) => setIsOverstyrBeregningChecked(e.target.checked)}
```

Comment:
Hvorfor bruke `onChange` her? Det skal vel RHF fikse for deg når man har registrert komponentet

Reply:
+1